components/demo/demo-snippet.js

```diff
@@ -68,6 +75,23 @@ class DemoSnippet extends LitElement {
 				position: sticky;
 				top: 0;
 			}
+			d2l-dropdown.settings-dropdown {
+				position: fixed;
+				top: -0.25rem;
```

Comment:
This probably should match the border radius units, which I think are in `px`. Is there a variable?
```suggestion
				top: -5px;
```

Reply:
Never mind. They do use `rem` and `-0.25rem` is about the limit before we start seeing the radius.